packaging/aws-cloudformation/firecamp.template

```diff
@@ -393,6 +393,14 @@
                             "Ref": "AppAccessSecurityGroup"
                         }
                     },
+                    {
```

Comment:
please add to firecamp-master.template as well. thanks!

Reply:
updated